yapapi/executor/_smartq.py

```diff
@@ -27,7 +27,16 @@
 Item = TypeVar("Item")
 
 
-class Handle(Generic[Item], object):
+class Handle(
+    Generic[Item], object
+):  # Handling? QueueItem? ConsumedItem? ConsumerItem? ConsumerBinding?
```

Comment:
I've got used to `Handle`, it's not that bad. Either way, let's not forget to remove this comment.